tests/plugins/contextmenu/integration.js

```diff
@@ -0,0 +1,113 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: clipboard,contextmenu */
+
+( function() {
+	'use strict';
+
+	bender.editor = true;
+
+	bender.test( {
+		'test getItemByCommandName - not rendered menu': function() {
+			assert.isNull( this.editor.contextMenu.findItemByCommandName( 'paste' ) );
+		},
+
+		'test getItemByCommandName - opened menu': function() {
+			bender.editorBot.create( {
+				name: 'editor_cm1'
```

Comment:
Does it make sense to name it like `editor_cm1`? Same value will be just naming it `editor1`, `editor2` etc.